src/serviceWorker.js

```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
```

Comment:
I'm unfamiliar with service workers - what does this do? / Is it used to build the production js?

Reply:
From my understanding, yes it assists with the production build.. and maybe some of the dev server capabilities... but I'm not 100% positive. We could research more documentation on this topic.